### ISSUE TITLE
The results of the experiment reproduce the problem

### ISSUE THREAD

**Issue by Lcode119** [zhou745/GauFuse_WSTAL#10]
Hello author. I trained it locally with your code and then tested it on the test set and the best result was 47.6%. What do I need to do to get the results in the paper?Looking forward to your reply, thank you!

**Comment by zhou745** [comment]
Hi, this is not a big gap, and it's caused by randomness. It relates to the numpy pytorch scipy version and your local machine system which introduces some randomness in the initialization state.  Because this problem highly relates to your local machine(which will also affect the performance of my baseline rskp, so the performance gain is more important), I suggest you try to adjust the gau_fuse temperature or inner-outer score ratio which should get you to a closer (even higher result).

**Comment by Lcode119** [comment]
Thank you for your reply, I'll try it again.
